packages/account-v2/webpack.config.js

```diff
@@ -0,0 +1,196 @@
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
```

Comment:
I'd suggest to split webpack config and move it also to TS, maybe we can do it later as well

Reply:
How do u recommend we split?

Reply:
I meant to move plugins, resolvers, loaders to separate files and get them with functions